doc/release-notes.md

```diff
@@ -106,11 +106,21 @@ Low-level changes
 Configuration
 ------------
 
-* An error is issued where previously a warning was issued when a setting in
+- An error is issued where previously a warning was issued when a setting in
   the config file was specified in the default section, but not overridden for
   the selected network. This change takes only effect if the selected network
   is not mainnet.
 
+Network
+-------
+
+- When fetching a transaction announced by multiple peers, previous versions of
```

Comment:
It'd be helpful to give a little more context and say what motivates the change. E.g. is it a bugfix? security improvement? efficiency improvement?

Reply:
It is a fix to the InvBlock bug.

Added a sentence.

Reply:
Thanks! I guess I'd also suggest describing the context in the first sentence instead of the last sentence to make the release notes easier to skim. But this is also fine in its current form.